src/pages/edit-profile/EditProfile.tsx

```diff
@@ -168,6 +169,8 @@ const EditProfile = () => {
     if (professionalBlock)
       dataToUpdate.professionalBlock = profileState.professionalBlock
 
+    if (aboutStudent) dataToUpdate.aboutStudent = aboutStudent
```

Comment:
Consider adding curly braces for better readability.

```suggestion
    if (aboutStudent) {
      dataToUpdate.aboutStudent = aboutStudent
    }
```